### PR TITLE
7.x

### PR DESCRIPTION
**Pull request by MorganDawe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ books.  The CWRCWriter only supports the Firefox browser.
 
 CONFIGURATION
 -------------
+The islandora_image_annotation module is a required module dependency.
```
Comment (willtp87) [inline README.md]: Probably don't need to mention a dependency listed in the .info in the configuration section of the readme.

File: cwrcwriter/resources/smoothness/jquery-ui-1.8.13.custom.css [hunk 2]
```diff
@@ -303,10 +303,10 @@
  */
 .ui-resizable { position: relative;}
 .ui-resizable-handle { position: absolute;font-size: 0.1px;z-index: 99999; display: block;
-	/* http://bugs.jqueryui.com/ticket/7233
-	 - Resizable: resizable handles fail to work in IE if transparent and content overlaps
-	*/
-	background-image:url(data:image/gif;base64,R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=);
+    /* http://bugs.jqueryui.com/ticket/7233
```
Comment (willtp87) [inline cwrcwriter/resources/smoothness/jquery-ui-1.8.13.custom.css]: If we can I think we should avoid modifying auto-generated files.

File: includes/callbacks.inc [hunk 3]
```diff
@@ -17,6 +17,7 @@ function islandora_critical_edition_setup(AbstractObject $fedora_object) {
   module_load_include('inc', 'islandora_basic_collection', 'includes/utilities');
   module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
   global $user;
+  $stroke_width = variable_get('annotation_stroke_width', '1%');
```
Comment (willtp87) [inline includes/callbacks.inc]: We should rename this variable to follow standards while we still can.

File: theme/islandora_critical_edition.tpl.php [hunk 4]
```diff
@@ -91,12 +95,13 @@
                 <div id="canvas-body-wrapper" style="width: 100%; height: 800px;"><div id="canvas-body">
 
                     <ul class="menu_body" id="show_body">
+                      <!-- TODO: Remove this unordered list (jplayer)? -->
```
Comment (willtp87) [inline theme/islandora_critical_edition.tpl.php]: Can we do?

File: theme/theme.inc [hunk 5]
```diff
@@ -13,12 +13,22 @@
 function islandora_critical_edition_preprocess_islandora_critical_edition(&$variables) {
   module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
   module_load_include('inc', 'islandora_critical_edition', 'includes/utilities');
+
+  // Local Critical Editions module path.
```
Comment (willtp87) [inline theme/theme.inc:21]: The ambiguity might warrant a variable re-name.

File: theme/theme.inc [hunk 6]
```diff
@@ -42,25 +53,36 @@ function islandora_critical_edition_preprocess_islandora_critical_edition(&$vari
...
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/jquery.rdfquery.rdfa.min-1.1.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/jquery.rdf.turtle.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/jquery.touchSwipe-1.2.4.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/raphael.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/scale.raphael.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/uuid.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/js/jquery/jquery.miniColors.js');
+  drupal_add_js($image_annotation_module_path . '/shared_canvas/stable/dialog_annotate.js');
+
+  // ~~~~~~~~~~~~~ TODO: LIB this stuff up nicly..
```
Comment (willtp87) [inline theme/theme.inc]: We should definitely de-dupe.

File: theme/theme.inc [hunk 7]
```diff
@@ -13,12 +13,22 @@
...
 
-  $variables['images_path'] = url($module_path . '/cwrcwriter/resources', array('absolute' => TRUE));
+  // Use the theme files from image annotation for annotation list
+  // and image annotation pane's.
+  $variables['anno_list_pane'] = theme('islandora_anno_list_pane');
+  $variables['anno_img_pane'] = theme('islandora_anno_image_pane');
+
+  // Fill additional session variables.
+  $variables['images_path'] = url($image_annotation_module_path . '/shared_canvas', array('absolute' => TRUE));
   $variables['viewer_url'] = url($module_path .
```
Comment (willtp87) [inline theme/theme.inc]: No longer being used?

File: theme/theme.inc [hunk 8]
```diff
@@ -13,12 +13,22 @@
...
   $pids = array_keys($pages);
 
-  $variables['images_path'] = url($module_path . '/cwrcwriter/resources', array('absolute' => TRUE));
+  // Use the theme files from image annotation for annotation list
+  // and image annotation pane's.
+  $variables['anno_list_pane'] = theme('islandora_anno_list_pane');
+  $variables['anno_img_pane'] = theme('islandora_anno_image_pane');
+
+  // Fill additional session variables.
+  $variables['images_path'] = url($image_annotation_module_path . '/shared_canvas', array('absolute' => TRUE));
```
Comment (willtp87) [inline theme/theme.inc:35]: New variables should be documented in the template.

**Comment by willtp87** [comment]
/cwrcwriter/resources/impl is still around do we need anything in the res directory? -- Morgan took care of this.

**Comment by willtp87** [comment]
Can we rename islandora_emic_init.js?

EDIT(Morgan): I have renamed the file islander_init.js.

**Comment by willtp87** [comment]
Should we move the behaviors for this module and the other into the respective startup scripts?
EDIT(Morgan): I have taken this suggestion and moved the window behaviours in to the respected modules cwrc/emic startup script.

**Comment by willtp87** [comment]
Looks like the middle pane stopped re-sizing with the draggable for me.  Can also independently resize that pane now.  These issues were pre-existing.

**Comment by willtp87** [comment]
Full window seems broken.

Fixed in latest commit.

**Comment by willtp87** [comment]
Image annotation list is unresponsive after changing pages and going back again.
This issue was preexisting there may be a better way to tackle this now: https://github.com/discoverygarden/islandora_critical_edition/pull/21

File: theme/theme.inc (hunk 8 above)
Comment (MorganDawe) [inline theme/theme.inc:35]: I have udpated the variables in the template.

Comment (willtp87) [inline theme/theme.inc:35]: You noted them in the preprocessor, look at the template: https://github.com/MorganDawe/islandora_critical_edition/blob/7.x/theme/islandora_critical_edition.tpl.php#L7

Comment (MorganDawe) [inline theme/theme.inc:35]: The variables have been updated in the template as well now.

File: theme/islandora_critical_edition.tpl.php [hunk 9]
```diff
@@ -5,6 +5,8 @@
  *
  * Available variables:
  * - $viewer_url (string): The url to the cwrc viewer.
```
Comment (willtp87) [inline theme/islandora_critical_edition.tpl.php]: No longer used.

**Comment by willtp87** [comment]
The two CSS files https://github.com/MorganDawe/islandora_critical_edition/blob/045c56663761d812429405378bbb7112b8999aa7/theme/theme.inc#L66-L67 can be put in a library function similar to and for the same reasons as the JS files.

**Comment by willtp87** [comment]
We should look into these two similarly named files https://github.com/MorganDawe/islandora_critical_edition/blob/045c56663761d812429405378bbb7112b8999aa7/theme/theme.inc#L61 and https://github.com/MorganDawe/islandora_image_annotation/blob/d3320cd10371f8a49a17f0ab337df2c7fbbc7d1e/islandora_image_annotation.module#L203

**Comment by willtp87** [comment]
The two CSS files of different versions may be unnecessary and could be troublesome in the future: https://github.com/MorganDawe/islandora_image_annotation/blob/d3320cd10371f8a49a17f0ab337df2c7fbbc7d1e/islandora_image_annotation.module#L201 and https://github.com/MorganDawe/islandora_critical_edition/blob/045c56663761d812429405378bbb7112b8999aa7/theme/theme.inc#L59 perhaps we can use the one in image annotation only? if so perhaps it could be included via https://github.com/discoverygarden/islandora_critical_edition/pull/23#issuecomment-20739290

**Pull request merged**